### DISCUSSION TITLE
Theme in sub folder and get_template_directory_uri()

### DISCUSSION THREAD

**Discussion by Aetles** [gregsullivan/_tw#109] [General]
It's quite clever that you can put the entire _tw generated _my-custom-generated-theme_ folder inside `wp-content/themes/` for you local development and that it works even though the actual theme is in the `theme` subfolder. 

But I ran into problem with using `get_template_directory_uri()` since it will output the path of theme folder **and** the subfolder (like `my-custom-generated-theme/theme`) and that breaks my paths locally. Is there any clever way around it? (Preferably without have to add dev/prod specific code.) 

Otherwise I think I'll stick to have the main _my-custom-generated-theme_ folder outside of WordPress structure and symlink the `theme` folder.

**Comment by gregsullivan** [comment]
Hi Adrian—Thanks for posting this!

Can you describe how it breaks your paths locally? I haven't run into any problems with it to date, but I'd love to hear what you're encountering so we can try to sort it out.

**Comment by Aetles** [comment]
Oh no, it was just a typo, my bad 🙈 

The image I was trying to load was broken and I just assumed it was a path issue but all the time it was `<img scr=` instead of `<img src=` 🙄 It's late here in Sweden and I'm obviously too tired to code 🥱 

Thanks for the quick answer thought and as ever, great work with the theme! 😊

**Reply by gregsullivan** [reply]
No problem at all—very happy to hear the issue is now resolved, and that you're enjoying _tw!
